frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/projects/obligations.jspf

```diff
@@ -131,7 +137,9 @@
                 <th class="org-more-info">
                     <span title="<liferay-ui:message key="expand.all" />" data-show="false">&#x25BA</span>
                 </th>
-                <th><liferay-ui:message key="organisation.obligation" /></th>
+                <th>
+                    <span title="<liferay-ui:message key="ObligationLevel-ORGANISATION_OBLIGATION" />" data-show="false"><liferay-ui:message key="organisation.obligation" /></span>
```

Comment:
any significance of the attribute `data-show="false"` ?
please remove it from all the 4 `span`, if it's not used.

Reply:
@akapti Thanks for the review and test, removed the unwanted code, Please check.